content/_policy/accessibility._en.md

```diff
@@ -25,7 +25,8 @@ Login.gov is regularly evaluated through a series of semi-automated evaluation t
 
 Our completed accessibility assessment reports can be found below:
 
-* [IAL 1 Account creation report](/docs/accessibility-assessment-ial1-account-creation.pdf)
+* Download the [Account creation report [DOCX, 154KB]](/docs/2024-05-15_VPAT2.5Rev508-Identity-Authentication.docx)
```

Comment:
IIRC one of the recent Executive Orders or OMB guidances recommended converting content to HTML. I tried out pandoc to convert the `.docx` and it did a pretty decent job, would we consider an HTML version of one of these?

```bash
pandoc --from=docx --to=gfm docs/2024-05-15_VPAT2.5Rev508-Identity-Authentication.docx
```
(`gfm` because the default `markdown` format has multi-line tables that kramdown doesn't like)

Reply:
@zachmargolis that's a good call out. thank you for the example tool and command.

I mentioned it in a meeting and I think the priority/goal is to make those changes in another epic.